tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/signup_info.cpp

```diff
@@ -92,6 +92,7 @@ tcf_err_t SignupInfo::DeserializeSignupInfo(
     return presult;
 }
 
+/*
```

Comment:
Can we remove the functions not needed ?  Please add a comment in code why is this commented

Reply:
Ok, will remove